### PR TITLE
Add tests for PayloadHelper

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

File: modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+public class PayloadHelperTest extends TestCase {
+    @Test
+    public void testGetPayloadType() throws Exception{
+        SOAP11Factory factory = new SOAP11Factory();
+        SOAPEnvelope envelope = factory.getDefaultEnvelope();
+        OMElement element = AXIOMUtil.stringToOM("<name><value>Test</value></name>");
+        envelope.getBody().addChild(element);
+        int type = PayloadHelper.getPayloadType(envelope);
+        Assert.assertEquals(0, type);
```
Comment (manoramahp) [inline modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java]: Assert message is missing

File: modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    @Test
+    public void testSetXmlPayload() throws Exception{
+        SOAP11Factory factory = new SOAP11Factory();
+        SOAPEnvelope envelope = factory.getDefaultEnvelope();
+        OMElement element = AXIOMUtil.stringToOM("<name><value>Test</value></name>");
+        PayloadHelper.setXMLPayload(envelope, element);
+        Assert.assertEquals("name", envelope.getBody().getFirstElementLocalName());
```
Comment (manoramahp) [inline modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java]: Assert message is missing

File: modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java (hunk 1 above)
Comment (ThishaniLucas) [reply modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java]: done

File: modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java (hunk 2 above)
Comment (ThishaniLucas) [reply modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java]: done

File: modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import junit.framework.Assert;
+import junit.framework.TestCase;
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.util.AXIOMUtil;
+import org.apache.axiom.soap.SOAPEnvelope;
+import org.apache.axiom.soap.impl.llom.soap11.SOAP11Factory;
+import org.junit.Test;
+
+public class PayloadHelperTest extends TestCase {
```
Comment (nuwanw) [inline modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java]: missing class level java doc comment

Comment (ThishaniLucas) [reply modules/core/src/test/java/org/apache/synapse/util/PayloadHelperTest.java]: done

**Pull request merged**